models/User.go

```diff
@@ -747,7 +782,19 @@ func PlaceAskOrder(userId uint32, ask *Ask) (uint32, error) {
 		return 0, NotEnoughStocksError{currentAllowedQty}
 	}
 
-	l.Debugf("Check2: Passed. Creating Ask.")
+	l.Debugf("Check2: Passed.")
+	orderPrice := getOrderFeePrice(ask.Price, ask.StockId, ask.OrderType)
+	orderFee := uint64((ORDER_FEE_PERCENT / 100.0) * float64(ask.StockQuantity*orderPrice))
```

Comment:
Can you put this line into getOrderFeePrice function, else you can change function name as getOrderFeePrice() does not return fee

Reply:
Will do :+1: